docs/health/health_background/background.rst

```diff
@@ -25,9 +25,15 @@ just add `--health-bg-checks` to your `pyats run job` command::
     pyats run job <job file> --testbed-file <testbed file> --health-bg-checks cpu memory logging core
 
 For more advanced users, you are able to provide your own health yaml file. This can be provided with the `--health-bg-file` argument like so::
+URL with token can be given like below example.
 
     pyats run job <job file> --testbed-file <testbed file> --health-bg-file /path/to/health.yaml --health-bg-checks cpu memory logging core
-    pyats run job <job file> --testbed-file <testbed file> --health-bg-file "http://path.to/health.yaml" --health-bg-checks cpu memory logging core
+    pyats run job <job file> --testbed-file <testbed file> --health-bg-file "http://<url>/health.yaml" --health-bg-checks cpu memory logging core
+    pyats run job <job file> --testbed-file <testbed file> --health-bg-file "http://<token>@<url>/health.yaml" --health-bg-checks cpu memory logging core
```

Comment:
I believe this is the same command as the next one (line 36), intentional to have duplicate?

Reply:
good catch! it was duplicated. so removed.